CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Welcome to openshift-python-wrapper contributing guide
+
+Thank you for contributing to our project!  
+[Project maintainers](https://github.com/RedHatQE/openshift-python-wrapper/blob/main/OWNERS)
+
+## New contributor guide
+
+To get an overview of the project, read the [README](README.md).
+
+## Issues
+
+### Create a new issue
+If you find a problem with the code, [search if an issue already exists](https://github.com/RedHatQE/openshift-python-wrapper/issues).  
+If a related issue doesn't exist, you can open a new issue using a relevant [issue form](https://github.com/RedHatQE/openshift-python-wrapper/issues/new/choose).
```

Comment:
Worth to mention that if someone pushes PR then issues will be created automatically for the PR, so in this case no need to open issue.

Reply:
done

Reply:
`If you open a pull request to fix the problem, an issue will ba automatically created.`
should be before line 14

Reply:
done